code/modules/mob/emote.dm

```diff
@@ -24,15 +24,20 @@
 					return target
 
 // All mobs should have custom emote, really..
-/mob/proc/custom_emote(var/m_type=1,var/message = null)
+/mob/proc/custom_emote(var/m_type=EMOTE_VISUAL,var/message = null)
 
 	if(stat || !use_me && usr == src)
 		if(usr)
 			to_chat(usr, "You are unable to emote.")
 		return
 
-	var/muzzled = istype(src.wear_mask, /obj/item/clothing/mask/muzzle)
-	if(m_type == 2 && muzzled) return
+	var/muzzled = is_muzzled()
+	if(muzzled)
+		var/obj/item/clothing/mask/muzzle/M = wear_mask
+		if(m_type == EMOTE_SOUND && M.mute)
```

Comment:
Does this mean that any value of `M.mute` will block sound emotes, instead of doing a mask check?

Reply:
Yes, in this case they are wearing a muzzle that blocks sound. Not sure if it should be, but will probably make it so it wont.